packages/osintgram/PKGBUILD

```
@@ -0,0 +1,40 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=osintgram
+pkgver=1.3.r5.g50d44fd
+pkgrel=1
+pkgdesc='OSINT tool offering an interactive shell to perform analysis on Instagram account of any users by its nickname.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon')
+url='https://github.com/Datalux/Osintgram'
+license=('GPL3')
+depends=('python' 'python-instagram-private-api' 'python-requests' 'python-requests-toolbelt' 'python-geopy' 'python-gnureadline' 'python-prettytable')
+makedepends=('git')
+source=("git+https://github.com/Datalux/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+package() {
+  cd $pkgname
+
```

```suggestion

  install -dm 755 "$pkgdir/usr/bin"
  install -dm 755 "$pkgdir/usr/share/$pkgname"

```

https://github.com/BlackArch/blackarch-pkgbuilds/blob/d57c25f19cd48b8a5af21d7948a509db8d9fdc92/PKGBUILD-generic-jointscript#L32-L33